README.md

```diff
@@ -35,15 +37,31 @@ Dependencies
 
 None
 
-Example Playbook
-----------------
+Example Playbooks
+-----------------
 
+Create a new VG named `new_vg` with two already attached disks, `/dev/sdc` and `/dev/sdd`.
```

Comment:
do you intend to setup `lvm_action=new_vg` into inventories or this is one-shot action (ie. we use extra-vars on command line or have a playbook into playbooks/oneshot/) ?

Reply:
As described here https://github.com/peopledoc/sre/pull/585 this will be a one-shot action.

Reply:
ok answer here https://github.com/peopledoc/peopleinfra/pull/4843/files